qcodes/config/__init__.py

```diff
@@ -1,2 +1,27 @@
+import json
+
 from .config import Config
 from .config import logger
+from .config import DotDict
+
+cfg = Config()
+loc = cfg['GUID_components']['location']
+station = cfg['GUID_components']['work_station']
+
+if station == 0 or loc == 0:
+    print('Welcome to QCoDeS!\nTo use QCoDeS, please provide the system with '
+          'enough information to generate unique dataset identifiers.')
+
+if loc == 0:
+    while loc == 0:
+        loc = int(input('Please enter an integer as your location code: '))
```

Comment:
is there maximum for the location number? if so, it needs to be mentioned and checked against.